pkgs/applications/misc/clight/clightd.nix

```
@@ -1,23 +1,31 @@
-{ lib, stdenv, fetchFromGitHub
+{ lib, stdenv, fetchFromGitHub, fetchpatch
 , dbus, cmake, pkg-config
-, glib, udev, polkit, libmodule
+, glib, udev, polkit, libusb, libjpeg, libmodule
```

`libusb` is an alias to `libusb1`.

```suggestion
, glib, udev, polkit, libusb1, libjpeg, libmodule
```